plugins/logging.js

```diff
@@ -52,9 +52,9 @@ function formatter(id, level) {
 }
 
 module.exports = function logging (server, conf) {
-  var level = conf.logging && conf.logging.level && LOG_LEVELS.indexOf(conf.logging.level) || DEFAULT_LEVEL
-  if (level === -1) {
-    console.log('Warning, logging.level configured to an invalid value:', conf.logging.level)
+  var level = conf.logging && conf.logging.level && LOG_LEVELS.indexOf(conf.logging.level)
+  if (level === undefined || level === -1 ) {
+    console.log('Warning, logging.level configured to an invalid value:', level)
```

Comment:
I think `level` should probably be `conf.logging.level` here now that I'm looking at it again.

`'Warning, logging.level configured to an invalid value: undefined'` doesn't seem to helpful.

In addition, not setting a log level will log this message, even though `conf.logging.level === undefined` is probably valid.

I can update this to reflect that thinking, but maybe I can get a sanity check real quick?